db/migrate/20170109045732_add_scores_to_sheets.rb

```diff
@@ -0,0 +1,6 @@
+class AddScoresToSheets < ActiveRecord::Migration
+  def change
+    add_column :sheets, :home_team_score_row, :integer, :array => true
+    add_column :sheets, :away_team_score_row, :integer, :array => true
```

Comment:
Ohhhh, didn't see that.  Never seen a datatype like that I always thought it had to be Enum if it was some sort of collection.